web_src/js/components/ContextPopup.vue

```diff
@@ -91,16 +91,22 @@ export default {
 <template>
   <div ref="root">
     <div v-if="loading" class="tw-h-12 tw-w-12 is-loading"/>
-    <div v-if="!loading && issue !== null">
-      <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
-      <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
-      <p>{{ body }}</p>
+    <div v-if="!loading && issue !== null" class="tw-flex tw-flex-col tw-gap-2">
+      <div class="tw-text-12">{{ issue.repository.full_name }} on {{ createdAt }}</div>
```

Comment:
```suggestion
      <div class="tw-text-12">{{ issue.repository.full_name }} - {{ createdAt }}</div>
```

(or use `@`)

If we use this trick, then maybe we do not need to translate it anymore ....

Reply:
Hmm, didn't we have some solution for `on` with `<relative-date>`?

Reply:
No. And it doesn't work for this case: "{something} on {somedate}"

Reply:
Ideally this text would be the same as in issue list:

<img width="181" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/ff9bea07-5d7c-4b03-940c-a38a49800af1">

But that'll be more work hmm.

Reply:
Then leave it as-is. Since it should be backported, avoid regression.

Reply:
Yeah let's leave it as is for now.